dashboard/serializers.py

```diff
@@ -25,6 +25,25 @@ def serialize_enrollments(cls, enrollments):
         return [{'title': course.title} for course in enrolled_courses]
 
     @classmethod
+    def count_courses_passed(cls, program, mmtrack):
+        """
+        Returns number of passed courses.
+
+        Args:
+            program (Program): A program object.
+            mmtrack (dashboard.utils.MMTrack): a instance of all user information about a program
+        Returns:
+            num_complete_courses: A number of passed courses.
+        """
+        num_courses_passed = 0
+        for course in program.course_set.all():
+            course_run = course.first_unexpired_run()
```

Comment:
It looks like I was wrong, `first_unexpired_run()` is not the function we should use here. Can you use `get_info_for_course` and get the status from the dict returned from that function? We should have only one place where the decision of whether a course is passed or not is made